portfolio/src/main/webapp/aboutme.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```

Comment:
Could you move this `<script>` tag to the bottom of the `<body>` tag?

This way, the full DOM will be available for scripting.

Reply:
Like under the </body> tag? On about line 40

Reply:
it should look like:
```
<body>
  <h1>Content</h1>
  <p>Lorem Ipsum... IDK</p>
  <script></script>
</body>
```

Reply:
Understood.

Reply:
Is there a specific reason we do this is or is it just for readability?